poetapp/frost-web
# adds Dockerfile

To test:
```bash
  $ docker build -t frost-web-test .
  $ docker run -p 4000:4000 -it frost-web-test
```
Open https://localhost:4000 in a browser and you should see the frost-web homepage.
